src/main/java/samza/examples/sql/samza-sql-groupby/src/main/sql/samza.sql

```diff
@@ -0,0 +1,5 @@
+insert into kafka.groupbyTopic
+  select GetPageKey(pv.requestHeader) as __key__, GetPageKey(pv.requestHeader) as pageKey, count(*) as Views
+  from kafka.tracking.`PageViewEvent` as pv
+  group by (GetPageKey(pv.requestHeader))
+-- Write additional SQL Code here
```

Comment:
Remove this line

Reply:
I added this line to emphasis that a sql file can contain multiple sql statements